gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/src/test/java/testcases/PlaceholderTestCase.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package testcases;/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.gravitee.apim.gateway.tests.sdk.AbstractGatewayTest;
+import io.gravitee.apim.gateway.tests.sdk.annotations.DeployApi;
+import io.gravitee.apim.gateway.tests.sdk.annotations.GatewayTest;
+import io.gravitee.apim.gateway.tests.sdk.policy.PolicyBuilder;
+import io.gravitee.apim.gateway.tests.sdk.policy.fakes.Header1Policy;
+import io.gravitee.apim.gateway.tests.sdk.policy.fakes.OnRequestPolicy;
+import io.gravitee.apim.gateway.tests.sdk.policy.fakes.Stream1Policy;
+import io.gravitee.apim.gateway.tests.sdk.policy.fakes.Stream2Policy;
+import io.gravitee.plugin.policy.PolicyPlugin;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.rxjava3.core.http.HttpClient;
+import java.util.Map;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@GatewayTest
+@DeployApi({ "/apis/placeholder.json" })
+@EnableForGatewayTestingExtensionTesting
+public class PlaceholderTestCase extends AbstractGatewayTest {
```

Comment:
TestCase are tests classes that are not executed during `mvn test` job. Those particular classes are used to test the behavior of the sdk, and do assertions on the expected output of a test class that has been executed with the sdk (x test passed, x failures, etc).

You have to manually call the TestCase as it is done here: https://github.com/gravitee-io/gravitee-api-management/blob/8062b631c25fc39e84ac1808a625122ccca89fbe/gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/src/test/java/GatewayTestingExtensionTest.java#L141